pulse/email-with-resend/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "prisma-pulse-resend",
+  "license": "MIT",
+  "scripts": {
+    "dev": "ts-node ./script.ts"
+  },
+  "dependencies": {
+    "@prisma/client": "5.12.1",
```

Comment:
should we bump prisma and prisma client versions?

Reply:
I think this _should_ be picked up by renovate once merged into `main`? https://github.com/prisma/prisma-examples/blob/latest/.github/renovate.json#L15

Reply:
@jharrell My assumption was that it should be handled by renovate as well.